test/src/main/java/org/apache/accumulo/test/performance/scan/CollectTabletStats.java

```diff
@@ -510,7 +510,7 @@ private static int readFilesUsingIterStack(VolumeManager fs, ServerConfiguration
     HashSet<ByteSequence> columnSet = createColumnBSS(columns);
 
     reader.seek(new Range(ke.getPrevEndRow(), false, ke.getEndRow(), true), columnSet,
-        columnSet.size() == 0 ? false : true);
+        columnSet.size() != 0);
```

Comment:
A lot of these could use `isEmpty()`

Reply:
Intellij has an inspection that will automatically fix this. I will run this later.